certified-connectors/Plumsail Documents/readme.md

```diff
@@ -0,0 +1,18 @@
+## Plumsail Documents Connector
+Plumsail Documents provides a powerful and very extensive REST API.  Using this API, you can generate documents in Microsoft Flow, Azure Logic Apps or PowerApps. 
```

Comment:
@RFlipper Please change the text from "Microsoft Flow" to "Microsoft Power Automate"

Reply:
@RFlipper Please change the text from "PowerApps" to "Power Apps" as well.